crates/datagen/src/lib.rs

```diff
@@ -245,6 +301,31 @@ fn parse_range_for_datetime(
     }
 }
 
+/// This is our max precision for decimals.
+const FELDERA_MAX_DECIMAL_PRECISION: u32 = 28;
+
+/// THis is our max scale for decimals.
+const FELDERA_MAX_DECIMAL_SCALE: u32 = 28;
+
+/// Given a precision and scale calculate the maximum Decimal value for it.
```

Comment:
I am not sure this is right.
It depends how you interpret the scale.
If the scale is the *actual* number of digits after the point, this is right.
But if it is the *maximum* number of digits after the point, then the max value is 10^p.

Reply:
good catch .. unfortunately scale in rust-decimal is different from the SQL scale :/